packages/core/src/shared/index.ts

```diff
@@ -30,6 +30,7 @@ export { AmazonqCreateUpload, Metric } from './telemetry/telemetry'
 export { getClientId, getOperatingSystem } from './telemetry/util'
 export { extensionVersion } from './vscode/env'
 export { cast } from './utilities/typeConstructors'
+export { openTextDocument } from './utilities/workspaceUtils'
```

Comment:
exporting specific functions from index.ts risks name conflicts in the future.

```suggestion
export * as workspaceUtils from './utilities/workspaceUtils'
```